web/app/js/components/ErrorModal.jsx

```diff
@@ -52,20 +52,21 @@ class ErrorModal extends React.Component {
 
     const byPodAndContainer = _map(errorsByPod, (podErrors, pod) => {
       const byContainer = _reduce(podErrors.errors, (errors, err) => {
+        const errors__ = errors;
```

Comment:
nit: could you use a single underscore, like you did in your other changes?

Reply:
Surely, I'll use a single underscore. :)